jar-infer/jar-infer-lib/src/test/java/com/uber/nullaway/jarinfer/JarInferTest.java

```diff
@@ -101,6 +103,24 @@ private void testJARTemplate(
     Assert.assertTrue("jar file not found! - " + outJARPath, new File(outJARPath).exists());
   }
 
+  private void testAnnotationInJarTemplate(
+      String testName,
+      String pkg,
+      String inputJarPath,
+      Map<String, String> expectedToActualAnnotationsMap)
+      throws Exception {
+    String outputFolderPath = outputFolder.newFolder(pkg).getAbsolutePath();
+    DefinitelyDerefedParamsDriver.reset();
```

Comment:
Might be better to rewrite `DefinitelyDerefedParamsDriver` to use instance methods rather than static, to get rid of this `reset()` method

Reply:
100% agreed. @navahgar, follow up PR for this?

Reply:
Sure. That is the plan. I didn't want to make more changes to this already big diff.